tensorflow/core/protobuf/config.proto

```diff
@@ -208,6 +208,9 @@ message GPUOptions {
     // size exceeds the requested memory by 5% of the total virtual device/gpu
     // memory size.
     double internal_fragmentation_fraction = 10;
+
+    // When true, use CUDA cudaMallocAsync API instead of TF gpu allocator.
+    bool use_cuda_malloc_async = 11;
   }
```

Comment:
Is this correct? Assigning an integer constant to a bool seems wrong.

Reply:
This is not assigning a value, but assigning an ID. So this is fine.